float32[] object_class
float32[] object_yaw
int32[] object_x
int32[] object_y
int32[] aruco_id
int32[] aruco_center_x
int32[] aruco_center_y


